bike-erp/backend/src/main/helpers/db_init.ts

```diff
@@ -20,25 +20,24 @@ export const initialize_db = (): void => {
         organization TINYTEXT NOT NULL
     );`;
 
-    const createComponentQuery: string = `
+  const createComponentQuery: string = `
     CREATE TABLE IF NOT EXISTS component(
         component_id int NOT NULL AUTO_INCREMENT PRIMARY KEY,
         price float(24) NOT NULL,
         quantity int NOT NULL DEFAULT 0,
         component_type ENUM('FRAME','WHEEL', 'SEAT','DRIVE_TRAIN', 'HANDLE') NOT NULL,
         component_status ENUM('AVAILABLE','UNAVAILABLE','INCOMING') NOT NULL,
         size ENUM('LARGE','MEDIUM','SMALL') NOT NULL,
-        specificComponentType TINYTEXT NULL
+        specific_component_type TINYTEXT NULL
```

Comment:
why did you not make it camel case. This will break some stuff since the code expect it in camel came

Reply:
Because look at component_type and component_status, it would not be consistent

Reply:
I agree, but this change will break other stuff. E.g. in the frontend i expect the attribute to called in camel case